core/templates/dev/head/pages/base.html

```diff
@@ -155,7 +155,7 @@ <h2>
         </div>
       </div>
 
-      <div ng-if="DEV_MODE" class="oppia-dev-mode">
+      <div ng-if="DEV_MODE" class="oppia-dev-mode" ng-cloak>
```

Comment:
Whatever changes you make in base.html, those should be replicated in https://github.com/oppia/oppia/blob/develop/core/tests/build_sources/templates/base.html.

Reply:
@abhaygupta97, done replicated it. Thanks!